html-forms/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="styles.css" href="styles.css">
+    <title>HTML-Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form method="post">
+      <label for="email">Email</label><br />
```

Comment:
instead of `<br />` tags, we should wrap the `label` element in a `div` (because `div`s are block elements).

Please do this everywhere else you are using a `<br />` tag on the page.